plugins/modules/system/vdo.py

```diff
@@ -258,6 +258,15 @@
               configured setting unless a different value is specified
               in the playbook.
         type: str
+    force:
+        description:
```

Comment:
I've taken the description from man page in
https://github.com/dm-vdo/vdo/blob/827e5e0974c62a4e169766020fb69b3f34015851/vdo-manager/man/vdo.8#L580-L584
Please change it in any way you see fit.